packages/theme_tailor_figma_loader/lib/src/utils/utils.dart

```diff
@@ -0,0 +1,51 @@
+import '../models/app_typography.dart';
+
+String toColor(String str) {
+  return 'Color(0xFF${str.substring(1, 7).toString().toUpperCase()})';
```

Comment:
This does not support colors with opacity (I guess we can add a method/extension on Color to load it from hex value and ust .toString() )